packages/cozy-procedures/src/redux/documentsSlice.js

```diff
@@ -0,0 +1,22 @@
+/*eslint no-unused-vars: off*/
+/*eslint no-console: off*/
+import { createSlice } from 'redux-starter-kit'
+
+const documentsSlice = createSlice({
+  initialState: null,
+  slice: 'documents',
+  reducers: {
+    init: (state, action) => {
+      console.warn('init documents slice not implemented')
```

Comment:
minilog

Reply:
Meh, those logs will be kicked out in the next couple of days.